.github/workflows/lighthouse.yml

```diff
@@ -27,12 +27,39 @@ permissions:
   pull-requests: write
 
 jobs:
-  lighthouse-ci:
+  get-vercel-preview:
     # We want to skip our lighthouse analysis on Dependabot PRs
```

Comment:
this comment needs to move with the ordering

Reply:
What do you mean? It's commenting on the line below (skipping the Dependabot).

Reply:
to me the comment nesting seems to suggest it is documenting the job directly below it, which is no longer lighthouse but get vercel preview.

it is probably just a small nit - sorry for noise